gettsim/social_insurance_contributions/eink_grenzen.py

```diff
@@ -15,23 +11,16 @@ def mini_job_grenze(wohnort_ost: BoolSeries, soz_vers_beitr_params: dict):
     -------
 
     """
-    out = wohnort_ost.replace(
-        {
-            True: soz_vers_beitr_params["geringfügige_eink_grenzen_m"]["mini_job"][
-                "ost"
-            ],
-            False: soz_vers_beitr_params["geringfügige_eink_grenzen_m"]["mini_job"][
-                "west"
-            ],
-        }
-    )
+    mini_job_params = soz_vers_beitr_params["geringfügige_eink_grenzen_m"]["mini_job"]
```

Comment:
```suggestion
    params = soz_vers_beitr_params["geringfügige_eink_grenzen_m"]["mini_job"]
```
As above. I like this shortening, but let's be consistent with it.